satpy/tests/writer_tests/test_mitiff.py

```diff
@@ -41,9 +41,10 @@ def tearDown(self):
 
     def _get_test_datasets(self):
         """Create a datasets list."""
-        import xarray as xr
-        import dask.array as da
         from datetime import datetime
```

Comment:
I know it's not from this PR, but just for my information — is there a good reason to have a python stdlib import repeated in every test rather than at the module level?